ansible/roles_ocp_workloads/ocp4_workload_portworx/tasks/pre_workload.yml

```diff
@@ -1,44 +1,70 @@
 ---
 
-- name: Gather aws security group info
-  amazon.aws.ec2_group_info:
+- name: Gather VPC information
+  amazon.aws.ec2_vpc_net_info:
+    region: "{{ aws_region }}"
+    aws_access_key: "{{ aws_access_key_id }}"
+    aws_secret_key: "{{ aws_secret_access_key }}"
+  register: vpc_info
+
+- name: Set VPC ID
+  set_fact:
+    vpc_id: "{{ vpc_info.vpcs[0].vpc_id }}"
+
+- name: Gather EC2 instances with "worker" in their Name tag
+  amazon.aws.ec2_instance_info:
     filters:
       "tag:Name": "*worker*"
     region: "{{ aws_region }}"
     aws_access_key: "{{ aws_access_key_id }}"
     aws_secret_key: "{{ aws_secret_access_key }}"
-  register: sg_info
+  register: ec2_info
+
+- name: Extract security group info with "node" in the group name
+  set_fact:
```

Comment:
Please use fully qualified module names. E.g. ansible.builtin.set_fact.

It would be good to run ansible-lint over the files to catch all of them.

Reply:
I ran ansible-lint on the files and updated the suggested items